include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 16
+#define SOF_ABI_MINOR 17
```

Comment:
how far are we with standardising our ABI changes? Haven't we yet agreed to first require a documented RFC for all changes?

Reply:
RFC as issue is good pre-work, we have work done here already so you can comment/ACK/NACK in PR

Reply:
@lyakh We agreed just that we need reviews from both sides, we don't have standarised flow for 'documenting', how would you like this to look? Author already wrote what he is bumping and why, however it may be not so 'detailed', cos he wrote that just backward compatibility is kept. Do you need it to be more detailed or maybe you need some other info?

Reply:
@jajanusz Having an implementation along with the ABI change is good IMHO. Without one, studying even an extended documentation might be difficult trying to understand whether the proposed change will really work. The change in this PR isn't particularly difficult either, so, I don't think it needs more documentation than is already provided. But the PR lacks both an ABI tag and an [RFC] keyword, so it could easily be missed.

Reply:
@lyakh @jajanusz  The new ABI process is still under discussion, it's not verified. Once we merge the sof-docs PR, then we can start following the new one, but until then, we do ABI changes with old rules.